packages/wonder-blocks-icon-button/components/icon-button-core.js

```diff
@@ -149,6 +149,9 @@ const sharedStyles = StyleSheet.create({
         // This removes the 300ms click delay on mobile browsers by indicating that
         // "double-tap to zoom" shouldn't be used on this element.
         touchAction: "manipulation",
+        ":focus": {
```

Comment:
Can you add a brief comment explaining what this is for? It's not immediately obvious for others who might look at this.